Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -3826,12 +3828,11 @@ bool CClientVehicle::HasPoliceRadio()
 
 void CClientVehicle::RemoveAllProjectiles()
 {
-    CClientProjectile*                 pProjectile = NULL;
-    list<CClientProjectile*>::iterator iter = m_Projectiles.begin();
-    for (; iter != m_Projectiles.end(); iter++)
+    CClientProjectile* pProjectile = nullptr;
```

Comment:
There is no need in dedicated `pProjectile` variable. You can just for `(auto* pProjectile : m_Projectiles)`